scale/scheduler/tasks/services/service.py

```diff
@@ -0,0 +1,126 @@
+"""Defines the abstract base class for all system services"""
+from __future__ import unicode_literals
+
+import logging
+from abc import ABCMeta, abstractmethod
+
+from job.tasks.update import TaskStatusUpdate
+
+
+logger = logging.getLogger(__name__)
+
+
+class Service(object):
```

Comment:
Didn't see any unit test coverage for this class.

Reply:
This is an abstract base class. It's concrete methods are tested in the unit tests for the messaging service.